### PR TITLE
Domains Fixes

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

- Fixed title typo
- Moved domains into the deploy section 
- Removed ssh dialogue from the domains deployment list

### Related Issues

- #3348
### Documentation PR

For UI changes, Please provide the Documetation PR on [info_grid](https://github.com/threefoldtech/info_grid)

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/App.vue [hunk 1]
```diff
@@ -317,6 +317,12 @@ const routes: AppRoute[] = [
         route: DashboardRoutes.Deploy.Applications,
         tooltip: "Deploy ready applications on the ThreeFold grid.",
       },
+      {
+        title: "Domains",
```
Comment (0oM4R) [inline packages/playground/src/App.vue:321]: should it be `Domains` or `Domain` ?

Comment (maayarosama) [reply packages/playground/src/App.vue:321]: According to a conversation with @AhmedHanafy725 in the past the solution name was to be changed from ```Expose``` to ```Domains```

File: packages/playground/src/components/vm_deployment_table.vue [hunk 2]
```diff
@@ -170,6 +170,7 @@ const props = defineProps<{
   projectName: string;
   modelValue: any[];
   deleting: boolean;
+  hideSsh?: boolean;
```
Comment (zaelgohary) [inline packages/playground/src/components/vm_deployment_table.vue]:
```suggestion
  hideSSH?: boolean;
```

**Review by zaelgohary — COMMENTED** [review]
Home has a link to all solutions in the sidebar. Can't we add a way to include domains too?

image


Health should be a chip

image

Backend keys could be just backends

image

File: packages/playground/src/components/vm_deployment_table.vue (hunk 2 above)
Comment (zaelgohary) [reply packages/playground/src/components/vm_deployment_table.vue]: hideSSH should change should be applied in all necessary places.

**Comment by maayarosama** [comment]
> Home has a link to all solutions in the sidebar. Can't we add a way to include domains too?
> 
> image
> 
> Health should be a chip
> 
> image
> 
> Backend keys could be just backends
> 
> image

Per discussion with @AhmedHanafy725, Adding domains to the home isn't necessary

**Review by zaelgohary — APPROVED** [review]

**Comment by amiraabouhadid** [comment]
Please resolve merge conflicts

**Review by Mahmoud-Emad — APPROVED** [review]
Good job ya Mayar

image


image

**Pull request merged**